core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -1141,14 +1141,88 @@ private void checkColumnUniquenessForFilterWithConstantColumns(String sql) {
         .assertThatAreColumnsUnique(bitSetOf(0, 1), is(false));
   }
 
+  @Test void testColumnUniquenessForLimit1() {
+    final String sql = ""
+                       + "select *\n"
+                       + "from emp\n"
+                       + "limit 1";
```

Comment:
Could we format this similar to other tests (e.g. remove empty string at beginning)? I know there are some other tests in this class like this, but from my experience it is not how most tests are formatted In Calcite

Reply:
I see that it is not consistent. There are quite a few like this in both this class and `RelOptRulesTest`. I did make the change everywhere and it was mostly fine with one exception - `testPullUpPredicatesFromUnion2`. In that case the SQL extends beyond the end of line and requires formatting that, IMO, makes it less pleasing.

I'd like to ask how strongly you feel about this one. One argument for leaving it as is is it makes all the lines of the SQL start on the same column, which I think is nice.

I will fix the issue with too much leading space on lines 1146-1148 and elsewhere. I must not have had my Code Style set properly when I wrote these tests.

Reply:
I can make the change but knowing where to stop is a bit tricky. Do I change just mine, all the ones in this file, all the ones in the files I changed, or all in all test files? I found 550 occurrences in Test, 372 in core:
<img width="485" alt="image" src="https://github.com/apache/calcite/assets/16478647/6e6e08dc-7abc-4855-957e-672ff60e209b">

Reply:
Feel free to leave it as is!

Reply:
Definitely don't go changing unrelated code. When a reviewer is asking for consistency, they are never asking for that.

There often inconsistencies when people have applied judgment. If there you can find reasons for those inconsistencies - even if they are reasons you don't personally agree with - it's not constructive to harp on those inconsistencies, because in so doing, you are criticizing people for applying judgment.

I would tend to write
```
String sql1 = "select x\n"
  + "from t";
String sql2 = ""
  + "select (select * from aVeryComplicatedSubqueryThatWouldMakeTheLineTooLong)\n"
  + from t"; 
```
There is a consistent rule there, if you look for it.